src/ui/js/Newgame.js

```diff
@@ -304,7 +353,13 @@ Newgame.formCreateGame = function() {
   var validSelect = true;
   var errorMessage;
 
-  if (!Newgame.activity.playerButton) {
+  if (!Newgame.activity.playerName) {
+    validSelect = false;
+    errorMessage = 'Please select the first player.';
+  } else if (Newgame.activity.playerName == Newgame.activity.opponentName) {
+    validSelect = false;
+    errorMessage = 'The two players cannot be the same person.';
```

Comment:
I would say the two players can't be the same username.  We have no way of knowing whether two players are the same person.  :>)

Reply:
Ah, good point.  :)